opennms-container/horizon/container-fs/confd/templates/newts.properties.tpl

```diff
@@ -0,0 +1,8 @@
+#
+# DON'T EDIT THIS FILE :: GENERATED WITH CONFD
+#
+org.opennms.newts.config.hostname={{getv "/opennms/cassandra/hostnames" "localhost"}}
+org.opennms.newts.config.keyspace={{getv "/opennms/cassandra/keyspace" "newts"}}
```

Comment:
I believe having Newts as the default is not good, especially because that is not OpenNMS's default. Why not using what we have (including the confd syntax) for all "conditional" stuff from the branch on which we've originally introduced confd?

Reply:
Disregard, the actual property to enable newts is on a different confd template. I should read everything first, and then comment ;)